src/main/webapp/recipe.jsp

```diff
@@ -5,29 +5,28 @@
 <t:genericpage>
     <jsp:body>
       <div class="content">
-        <h1>Recipe Title ${messages}</h1>
-        <img src="images/cover.png" class="img-fluid my-3" alt="Recipe Image">
+        <h1>${title}</h1>
+        <h3>by ${author}</h1>
+        <img src=${imgURL} class="img-fluid my-3" alt="Recipe Image">
         <ul class="list-group list-group-horizontal w-100">
-             <li class="list-group-item">Prep Time:   </li>
-             <li class="list-group-item">Cooking Time:   </li>
-             <li class="list-group-item">Difficulty:     </li>
+             <li class="list-group-item">Prep Time: ${prepTime}</li>
+             <li class="list-group-item">Cooking Time: ${cookTime}</li>
+             <li class="list-group-item">Difficulty: ${difficulty}</li>
         </ul>
         <br>
         <ul class="list-group w-25" >
-            <li class="list-group-item"> Ingredint 1</li>
-            <li class="list-group-item"> Ingredint 2</li>
-            <li class="list-group-item"> Ingredint 3</li>
+          <c:forEach items="${ingredient}" var="ingr">
```

Comment:
The same as the attribute inside the RecipeDetailsServlet, the list should be called `ingredients`.